src/api/dirmonitor.c

```diff
@@ -46,21 +46,19 @@ static int f_check_dir_callback(int watch_id, const char* path, void* L) {
     lua_pushnumber(L, watch_id);
 
   int result = 0;
-  if (lua_pcall(L, 1, 1, 3) == LUA_OK)
-    result = lua_toboolean(L, -1);
+  if (lua_pcall(L, 1, 1, 3) == LUA_OK) result = lua_toboolean(L, -1);
```

Comment:
I'm not sure we should keep this in a single line, it's a bit too long.
Same thing in a bunch of other places.